virtme_ng/run.py

```diff
@@ -874,23 +870,20 @@ def make(self, args):
             target = "bzImage"
             cross_compile = None
             cross_arch = None
-        make_command = MAKE_COMMAND
+        make_command = ["make", "LOCALVERSION=-virtme"]
         if args.compiler:
-            make_command += f" HOSTCC={args.compiler} CC={args.compiler}"
+            make_command += [f"HOSTCC={args.compiler}", f"CC={args.compiler}"]
         if args.skip_modules:
-            make_command += f" {target}"
+            make_command.append(target)
```

Comment:
Small nit and not related to your PR, but why is the order as it is? Normally, I would expect, first the target and then the makefile variables.

Reply:
Note sure, but I think it doesn't matter with the Makefile, you can run `make FOO=1 target -j48 BAR=2` I think

Reply:
It's easier to read IMO :) I would expect the arguments to be sorted a little bit at least... But yes, both should work.

Reply:
Sure, I can change that.